locales/ca/json.json

```diff
@@ -210,12 +210,12 @@
     "Editor": "Editor",
     "Editor users have the ability to read, create, and update.": "Els usuaris Editor tenen la capacitat de llegir, crear i actualitzar.",
     "Egypt": "Egipte",
-    "El Salvador": "Salvador",
+    "El Salvador": "El Salvador",
```

Comment:
Add the value to the [`locales/ca/_excludes.json`](https://github.com/Laravel-Lang/lang/blob/main/locales/ca/_excludes.json) file, otherwise it will show up as not translated.